python/ray/tune/log_sync.py

```diff
@@ -16,14 +20,25 @@
 # Map from (logdir, remote_dir) -> syncer
 _syncers = {}
 
+S3_PREFIX = 's3://'
+GCS_PREFIX = 'gs://'
+ALLOWED_REMOTE_PREFIXES = (S3_PREFIX, GCS_PREFIX)
 
 def get_syncer(local_dir, remote_dir=None):
     if remote_dir:
-        if not remote_dir.startswith("s3://"):
-            raise TuneError("Upload uri must start with s3://")
-
-        if not distutils.spawn.find_executable("aws"):
-            raise TuneError("Upload uri requires awscli tool to be installed")
+        if not any(remote_dir.startswith(prefix)
+                   for prefix in ALLOWED_REMOTE_PREFIXES):
+            raise TuneError("Upload uri must start with one of: {}"
+                            "".format(ALLOWED_REMOTE_PREFIXES))
+
+        if (remote_dir.startswith(S3_PREFIX)
```

Comment:
I believe gsutil can also rsync s3 paths, but maybe it's not a very common setup for AWS users.

Reply:
True, I guess they are interchangeable. Maybe leave it like this and see if there's need for change in the future.